config/available_locales.rb

```diff
@@ -5,8 +5,9 @@ module Sharetribe
   #
   # language format: ISO 639-1, two letters, lowercase
   # region format: ISO 3166, two letters, uppercase
-  # fallbacks: should not include US English, which is a last default fallback for each language
+  # fallbacks: should not include US English, which is a last default fallback for each langua
```

Comment:
Language, and actually, while you are at it, it should probably *the* last default fallback - not *a*.